apps/api/src/server/utils/mapping/map-case-status-string.js

```diff
@@ -0,0 +1,14 @@
+import { get } from 'lodash-es';
+
+/**
+ *
+ * @param {string} caseStatus
+ * @returns {string}
+ */
+export const mapCaseStatusString = (caseStatus) => {
+	const caseStatusMap = {
+		pre_application: 'Pre-Application'
```

Comment:
I was wondering whether the format of the statuses names would become a problem at some point.
maybe we want CaseStatus to have name, displayEn and displayCy?

Reply:
hmmmm good question. TBH i dont think Welsh is in plans for the MVP. @francescobrenci-AR ?

Reply:
oh, well, even without Welsh, isn't it better to have a `name` for being used in the code a displayName for the template?

Reply:
sorry don't quite get what you mean here @lutamKC

Reply:
oh sorry I forgot an "AND" and the sentence did not make sense.

i was just saying that since we're saving the status name as a regular text (with spaces, hyphens and stuff), we need things like this map above to use it in the code. so I was asking if it's worth to create an extra field in the caseStatus table that contains a property called `name` in which we put the "code-safe" name of the status.
which is exactly what we do with regions, zoom levels, sectors etc..

Reply:
Ah yes, so my intention is only to store the code-safe value of the status. The mapping will then post likely come from the code itself

Reply:
@mayya-bond-kc Welsh translation is not something we are doing for MVP but is happening at some point after that - I don't have a timeframe or requirements though. It's very much a WIP still